pkg/3scale/amp/operator/system_reconciler.go

```diff
@@ -22,6 +22,8 @@ type SystemReconciler struct {
 	*BaseAPIManagerLogicReconciler
 }
 
+var sts = false
```

Comment:
we could move this inside validateS3StorageProvidedConfiguration() function

Reply:
yes, sure, fixed. Thank you!

Reply:
yes, sure, fixed. Thank you!

Reply:
Hi @austincunningham , I attached file with details of testing done (sts and non sts, valid and not valid secrets)